### PR TITLE
complete PATCH /api/articles/:article_id endpoint and 400 error test …

### PR DESCRIPTION
**Pull request by BitKoda**
Completed ticket #7 (I think!) GET /api/articles/:id tests and endpoint -- card #7
Unsure if I needed more tests, but didn't think 404 was appropriate since the record would be first pulled from the db before updating???

### CODE REVIEW

File: __tests__/articles.test.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ beforeEach(() => seed(data));
 afterAll(() => connection.end());
 
 describe("GET /api/articles/:id ", () => {
-  it("status 200: returns a valid article", () => {
+  xit("status 200: returns a valid article", () => {
```
Comment (rogersop) [inline __tests__/articles.test.js]: Please don't commit skipped tests.

File: __tests__/articles.test.js [hunk 2]
```diff
@@ -48,3 +48,52 @@ describe("GET /api/articles/:id ", () => {
   });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
```
Comment (rogersop) [inline __tests__/articles.test.js:51]: I would recommend adding tests for trying to PATCH:
1. an article that doesn't exist
2. an invalid article ID
3. a valid article, but not including an `inc_votes` property on the object

**Review by rogersop — CHANGES_REQUESTED** [review]
Nice work on this one so far, tidy tests and code, just a couple other test cases to consider 😄

File: __tests__/articles.test.js (hunk 2 above)
Comment (BitKoda) [reply __tests__/articles.test.js:51]: Thanks for the feedback. I'll crack on with those in what's left of today/tomorrow.

File: __tests__/articles.test.js (hunk 1 above)
Comment (BitKoda) [reply __tests__/articles.test.js]: Done!

**Review by rogersop — APPROVED** [review]

**Pull request merged**